backend/src/main/java/ch/puzzle/okr/dto/checkIn/CheckInDto.java

```diff
@@ -0,0 +1,103 @@
+package ch.puzzle.okr.dto.checkIn;
+
+import ch.puzzle.okr.models.User;
+
+import java.time.LocalDateTime;
+
+public abstract class CheckInDto {
+    Long id;
+    String changeInfo;
```

Comment:
In der Beschreibung der Rest-Schnittstelle und im Klassendiagramm heisst das Feld `comment`. Was ist der Grund zur Umbenennung?

Reply:
Anstelle vom Kommentar wurde von @peggimann vorgegeben die "changeInfo" und die "initiatives" bestehen zu lassen, siehe #369